sdccc/src/test/java/com/draeger/medical/sdccc/util/MessageGeneratingUtilTest.java

```diff
@@ -187,7 +228,7 @@ private void verifySentRefs(final SoapMessage message, final List<String> refs)
         assertEquals(refs.size(), intersection);
     }
 
-    Envelope createSystemErrorReportWithRef(
+    com.draeger.medical.dpws.soap.model.Envelope createSystemErrorReportWithRef(
```

Comment:
Why did you remove the import and use the fully qualified name instead?

Reply:
That was not intentional. I was debugging a ClassCastException between two different Envelopes and forgot to revert the change afterwards. Thanks for spotting.